src/app/cli/src/cli_entrypoint/mina_cli_entrypoint.ml

```diff
@@ -1988,7 +1986,7 @@ let internal_commands logger ~itn_features =
             Prover.create ~commit_id:Mina_version.commit_id ~logger ~pids
               ~conf_dir ~proof_level
               ~constraint_constants:precomputed_values.constraint_constants
-              ~signature_kind:Mina_signature_kind.t_DEPRECATED ()
```

Comment:
Why sometimes you replace `Mina_signature_kind.t_DEPRECATED` with `Mina_signature_kind.Testnet` and some other times (like here) it is removed?

Reply:
We have this variable `signature_kind` in scope, so it's safe to refer to the one in scope instead of manual assign.